Pipfile

```diff
@@ -28,7 +28,7 @@ dpath = ">=1.5.0,<2"
 pyyaml = ">=5.4.1"
 boto3 = "==1.17.*"
 GitPython = "*"
-six = "==1.15.0"
+six = "==1.16.0"
```

Comment:
@tronxd Why is this required?

Reply:
@nimrodkor It has a conflict in pipenv when trying to install with 1.15 (guess that six 1.16 is pinned in one of our deps)